test/Microsoft.AspNet.Mvc.Core.Test/SessionStateTempDataProviderTest.cs

```diff
@@ -186,7 +187,7 @@ public void SaveAndLoad_SimpleTypesCanBeStoredAndLoaded()
                 { "string", "value" },
                 { "int", 10 },
                 { "bool", false },
-                { "DateTime", new DateTime() },
+                { "DateTime", inputDatetimeVal },
                 { "Guid", inputGuid },
                 { "List`string", new List<string> { "one", "two" } },
```

Comment:
This test is really 10 tests. Think about converting to a theory or just breaking it up.